runtime/doc/options.txt

```diff
@@ -1506,6 +1506,28 @@ A jump table for the options with a short description can be found at |Q_op|.
 	]	tag completion
 	t	same as "]"
 	f	scan the buffer names (as opposed to buffer contents)
+	F{func}	call the function {func}.  Multiple "F" flags may be specified.
```

Comment:
Are the results from multiple functions (providers) "merged"? Presumably it's chronological, and possibly re-stored as results from each arrive?

Does this resolve https://github.com/neovim/neovim/issues/32123 ?

Reply:
It seems to work:
```vim
func F1(findstart, base)
  if a:findstart
    return col(".") - 1
  endif
  return ["one", "two", "three"]
endfunc

func F2(findstart, base)
  if a:findstart
    return col(".") - 1
  endif
  return ["four", "five", "six"]
endfunc

set complete+=FF1,FF2
```
However the sorting seem to be a bit strange and is different depending on whether CTRL-N or CTRL-P is pressed. Probably needs some change.